application.js

```diff
@@ -18,27 +18,46 @@ $(document).ready(function(){
     });
   }
 
-  var filterSelection = function(){
-    response.tasks.forEach(function (task) {
-      if (task == all){
-        $('#todo-list').append('<div class="row"><p class="col-8 mt-3">' + task.content + '</p><div class="col-4"><button class="delete" data-id="' + task.id + '">Delete</button><input type="checkbox" class="mark-complete" data-id="' + task.id + '"' + (task.completed ? 'checked' : '') + '></div>');
+  var filterSelection = function(option){
```

Comment:
added the option parameter so you can pass something to the function telling it which button was clicked.